### DISCUSSION TITLE
[Question]: 2*A100部署qwen2.5-72B-instruct的问题

### DISCUSSION THREAD

**Discussion by lxb0425** [QwenLM/Qwen3#977] [Q&A]
### Has this been raised before?

- [X] I have checked [the GitHub README](https://github.com/QwenLM/Qwen2.5).
- [X] I have checked [the Qwen documentation](https://qwen.readthedocs.io) and cannot find an answer there.
- [X] I have searched [the issues](https://github.com/QwenLM/Qwen2.5/issues?q=is%3Aissue) and there is not a similar one.
- [X] I confirm that this is not a bug report, a feature request, or a badcase.

### Description

你好 使用vllm框架 默认配置 
2*A100部署qwen2.5-72B-instruct的并发大概是多少啊

**Comment by njzfw1024** [comment]
200左右把
